models/User.php

```diff
@@ -1,153 +1,102 @@
 <?php 
-// Permet d'avoir accès à la base de données
-require_once "Model.php";
 
 // Cette classe sert à manipuler tout ce qui touche aux users
-class User extends Model{
-    // Fonction permettant de vérifier que la session est bien active sous le role administrateur
-    function admin(){
-        if(isset($_SESSION['admin'])){
-            $tableau = [
-                'email'     =>  $_SESSION['admin'],
-                'role'      =>  'admin'
-            ];
-
-            $sql = "SELECT * FROM admins WHERE email=:email AND role=:role";
-            $req = $this->db->prepare($sql);
-            $req->execute($tableau);
-            $exist = $req->rowCount($sql);
-
-            return $exist;
-        }else{
-            return 0;
-        }
+class User
+{
+    private $id;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid variables with short names like $id. Configured minimum length is 3.](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5379676)